src/App.Metrics.Extensions.Reporting.ElasticSearch/Extensions/MetricValueExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Allan Hardy. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using App.Metrics.Apdex;
+using App.Metrics.Histogram;
+using App.Metrics.Meter;
+
+namespace App.Metrics.Extensions.Reporting.ElasticSearch.Extensions
+{
+    internal static class MetricValueExtensions
+    {
+        public static void AddApdexValues(this ApdexValue apdex, IDictionary<string, object> values)
+        {
+            values.Add("samples", apdex.SampleSize);
+            values.Add("score", apdex.Score);
+            values.Add("satisfied", apdex.Satisfied);
+            values.Add("tolerating", apdex.Tolerating);
+            values.Add("frustrating", apdex.Frustrating);
+        }
+
+        public static void AddHistogramValues(this HistogramValue histogram, IDictionary<string, object> values)
+        {
+            values.Add("samples", histogram.SampleSize);
```

Comment:
Added `sum`. I also refactored those extensions to mach Influx - another common area.